jitpack.yml

```diff
@@ -0,0 +1,7 @@
+# NOTE: it seems this file is not picked up by jitpack.iio, the echo does not show up in the build log
```

Comment:
Note: when we stop using jitpack for publishing and moveto maven central, any reference to Jitpack can onbviously be removed